package.json

```diff
@@ -3,23 +3,28 @@
   "version": "0.0.1",
   "description": "HTTP service to log monitoring information to StatsD",
   "main": "app.js",
+  "engines": {
+    "node": ">= 10.11.0",
+    "npm": ">= 6.4.1"
```

Comment:
Maybe add `yarn` here 🦁

Reply:
We don't really have the control of `yarn` version in travis. That's why I prefered not to set this/

Reply:
you can choose you version of yarn in travis, but yes it's ugly ...

Reply:
But I don't think about travis, I think about user here 😄